.env.development

```diff
@@ -10,5 +10,7 @@ SEGMENT_KEY=''
 DESIGNER_BASE_URL='http://designer.app:18808'
 HOSTNAME='example.com'
 UNBRANDED_LANDING_PAGE=''
-USE_MOCK_DATA=''
+USE_MOCK_DATA=true
```

Comment:
Thanks for cleaning up the test data -- could you comment like you did with the slug?